crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/model/KycRetrieveResponse.kt

```diff
@@ -10,20 +14,22 @@ import kotlinx.serialization.Serializable
  * This represents the exact structure expected by the Stripe API.
  */
 @Serializable
+@Parcelize
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-internal data class KycRetrieveResponse(
+data class KycRetrieveResponse(
     @SerialName("first_name")
-    val firstName: String,
+    override val firstName: String,
     @SerialName("last_name")
-    val lastName: String,
+    override val lastName: String,
     @SerialName("id_number_last4")
-    val idNumberLastFour: String?,
+    override val idNumberLastFour: String?,
     @SerialName("id_type")
     val idType: String?,
     @SerialName("dob")
-    val dateOfBirth: DateOfBirth,
+    @Serializable(with = DateOfBirthSerializer::class)
+    override val dateOfBirth: DateOfBirth,
 
     @SerialName("address")
     @Serializable(with = PaymentSheetAddressSerializer::class)
-    val address: PaymentSheet.Address
-)
+    override val address: PaymentSheet.Address
+) : Parcelable, VerifyKYCInfo
```

Comment:
`VerifyKYCInfo` is a view-layer interface that shouldn't be applied to data models. We should be using converter methods instead, e.g. 
```kotlin
fun KycRetrieveResponse.toVerifyKYCInfo(): VerifyKYCInfo
```